service-deployment/bootstrap/pylon/pylon.yaml.template

```diff
@@ -37,12 +37,16 @@ spec:
           env:
             - name: REST_SERVER_URI
               value: {{ clusterinfo['pyloninfo']['rest_server_uri'] }}
-            - name: PROMETHEUS_URI
-              value: {{ clusterinfo['pyloninfo']['prometheus_uri'] }}
             - name: K8S_API_SERVER_URI
               value: {{ clusterinfo['pyloninfo']['k8s_api_server_uri'] }}
             - name: WEBHDFS_URI
               value: {{ clusterinfo['pyloninfo']['webhdfs_uri'] }}
+            - name: PROMETHEUS_URI
```

Comment:
>PROMETHEUS_URI [](start = 20, length = 14)

please update readme accordingly. grafana is missing. And k8s dashboard are currently named as k8s_api_server_uri.

Reply:
Thanks Fan. Done.